src/config/errors.ts

```diff
@@ -0,0 +1,3 @@
+export const EMAIL_ERROR_MESSAGE =
```

Comment:
Have this as an enum and put this code in the types/data.ts file

Reply:
thanks, resolved